app/components/heading.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Components
```

Comment:
@SethHorsley , i didn't got why you are ejecting the typography components

Reply:
I ejected them because i didnt want to change the styling of the site so much and have to make the customizations in every componant instance of heading or text.